templates/default/delos.css

```diff
@@ -5910,7 +5910,7 @@ button.close {
   height: 10px;
   margin: 1px;
   text-indent: -999px;
-  border: 1px solid #ffffff;
+  border: 1px solid #fff;
```

Comment:
In my opinion this is not a 'no change'. It's an small optimization, since the new line is shorter (this is similar to your proposal to fix the indentation above, the current version does not break anything, it's just not quite there yet). I would not do the picky solution, but agree that we should check to use to same options for lessc. If I am not completely off, using the same version without special configs should do the trick. Obviously somebody is committing the opposit. This is a problem, and you are right that we should tackle this on a JF.